README.md

```diff
@@ -1,71 +1,46 @@
-Terraform Provider for Google Cloud Platform
-==================
+# Terraform Provider for Google Cloud Platform (Beta)
 
-- Website: https://www.terraform.io
+- Tutorials: [learn.hashicorp.com](https://learn.hashicorp.com/terraform?track=getting-started#getting-started)
+- Forum: [discuss.hashicorp.com](https://discuss.hashicorp.com/c/terraform-providers/tf-google/)
 - Documentation: https://www.terraform.io/docs/providers/google/index.html
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
-Maintainers
------------
+The Terraform Google provider beta is a plugin that allows [Terraform](https://www.terraform.io) to manage resources on Google Cloud Platform. This provider is maintained by the [Terraform team at Google](https://cloudplatform.googleblog.com/2017/03/partnering-on-open-source-Google-and-HashiCorp-engineers-on-managing-GCP-infrastructure.html) and the Terraform team at [HashiCorp](https://www.hashicorp.com/)
 
-This provider plugin is maintained by:
+This is the ['google-beta' provider](https://github.com/hashicorp/terraform-provider-google-beta) which contains all the features in ['google' provider](https://github.com/hashicorp/terraform-provider-google) as well as the preview features and features at a beta [launch stage](https://cloud.google.com/products#product-launch-stages). See [Provider Versions](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/guides/provider_versions) for more details on how to use `google-beta`.
 
-* The [Google Cloud Graphite Team](https://cloudplatform.googleblog.com/2017/03/partnering-on-open-source-Google-and-HashiCorp-engineers-on-managing-GCP-infrastructure.html) at Google
-* The Terraform team at [HashiCorp](https://www.hashicorp.com/)
+## Quick Starts
 
-Requirements
-------------
+- [Getting Started with the Google Provider](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/guides/getting_started)
+- [Provider Documentation](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs)
 
-- [Terraform](https://www.terraform.io/downloads.html) 0.12+
+## Provider Usage
 
-Building The Provider
----------------------
+Please see [instructions](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference) on how to configure the Google Provider.
 
-Clone repository to: `$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta`
+### Using local Provider
```

Comment:
I'd be inclined to omit this header- local providers are mostly used when contributing or forking the provider, and folks will find that documentation by looking for contribution materials.

Reply:
Make sense! Removed the section.